### ISSUE TITLE
what happens if the list is empty wouldnt this throw ?

### ISSUE THREAD

**Issue by falfaddaghi** [AyhamRatrout/Stack#2]
https://github.com/AyhamRatrout/Stack/blob/dca7d292b828158415e4b13fd32ff52635d984e4/src/Stack_Classes/LinkedList.cs#L44

**Comment by AyhamRatrout** [comment]
Yes it would. With the way I designed the stack, I had the pop method handle the situation in which the list would be empty because I wanted the design to print messages instead of throwing exceptions. With that said, I can have the method itself throw the exception is that's considered a better practice?
